<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateDataDialog</class>
 <widget class="QDialog" name="UpdateDataDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>227</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Data</string>
  </property>
  <widget class="QLabel" name="PrimaryKeyLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Primary Key:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>150</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PrimaryKeyInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="ColumnsOptions">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ValueInput">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ColumnLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Column:</string>
   </property>
  </widget>
  <widget class="QLabel" name="ValueLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Value:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
